_episodes/01-short-introduction-to-Python.md

```diff
@@ -63,6 +63,21 @@ Hello World
 * "Scripting" Mode: executing a series of "commands" saved in text file,
   usually with a `.py` extension after the name of your file:
 
+Use a text editor to create a file called `my_script.py`:
+
+Common text editors you can use: `vi`, `emacs`, `nano`, TextEdit, Spyder, PyCharm
+
+Open your favourite text editor, and type the following into the document:
+
+~~~
+print("Hello World")
+~~~
+{: .language-python}
+
+Save the file as `my_script.py`.
```

Comment:
```suggestion
For example, you could create a new file using your favourite text editor,
and type the following into the document:

~~~
print("Hello World")
~~~
{: .language-python}

(Common text editors you can use: `vi`, `emacs`, `nano`, TextEdit, Spyder, and PyCharm.)

Save the file as `my_script.py`.
```

Reply:
Thank you for your contribution and implementing one of @tobyhodges comments, @acbuyan. Is there a specific reason why you didn't use this suggestion as well?